### ISSUE TITLE
Deemphasize tag use

### ISSUE THREAD

**Issue by laurencelundblade** [ietf-rats-wg/draft-ietf-rats-msg-wrap#149]
There's general agreement that tags are only in CMW because some early implementations use them. If we were writing the document today we would leave out section 3.2 and focus solely on content format and media type.

Please remove mention of tags in the abstract and introduction.

Please note that section 3.2 is not recommended practice.

**Comment by thomas-fossati** [comment]
@laurencelundblade can we close this?

**Comment by nedmsmith** [comment]
Makes sense to close it but I still don't agree that restricting CMW to only TN tags was necessary or desriable.
